### DISCUSSION TITLE
pivpn stop working....

### DISCUSSION THREAD

**Discussion by tleveque23** [pivpn/pivpn#1715] [Support]
Hi, I don't want to open a bug report yet... maybe it's me...
Last month, I installed pivpn with Wireguard on the same rasberry pi than PiHole. I followed the instructions and everything worked well. I was able to connect using my iPhone and all the ads were blocked by PiHole as expected.
I did not use it for a week, then suddenly today, it is not working anymore. It is connecting without problems. Wireguard does not report any problem. But impossible to ping or resolve any addresses. Nothing is happenig.
Since last week, two thing happened: An iOs update. And a 24h power outage.
I don't know know what to do now. All the troubleshooting I found is about connection problems.
What I tried so far: 
- reboot the raspberry Pi
- restart the service
- check the debug (pivpn debug)
- check my external ip and check that DuckDns is up to date.
- check my port forwarding on my router
- delete the client and create a new one (and import it on my phone of course)

Any idea what to check for? What to do?
Thanks

**Comment by orazioedoardo** [comment]
Please perform troubleshooting steps https://docs.pivpn.io/faq/#how-do-i-troubleshoot-connection-issues

**Comment by tleveque23** [comment]
I found the problem. In my setupVars.conf file, my pivpnHOST did not contain my duckdns name. It was just the ip. Probably an old ip...
What is confusing is that WireGuard and the iPhone do not complain and act like if the connection to the vpn is fine. No error message and VPN at the top of the screen.
Don't believe that. Check your config file!
